### ISSUE TITLE
HTR Model Overview Chart: HTR+ best net stored annotation has wrong x-value in some cases

### ISSUE THREAD

**Issue by kahlep** [Transkribus/TranskribusSwtGui#290]
The min. value determined by the legacy HTR routine on the validation set CER log does not always match the best net stored by tensorflow.

Mostly, the difference is just one (e.g. HTR 16523: swt gui 13, tensorflow 14)
but in some cases there is a huge error (e.g. HTR 12425: swt gui 344, tensorflow 386) 

Store correct value in DB for all models and do not try to determine this in the client.

**Comment by kahlep** [comment]
If a base model was used during training, the model_best* files in some cases belongs to the base model, e.g. HTR 16464 in comparison with HTR 12425.
How to detect and display this?
